maui-toolkit-toc.html

```diff
@@ -245,6 +245,7 @@
             <li><a href="/maui-toolkit/NumericEntry/Basic-Features">Basic Features</a></li>
             <li><a href="/maui-toolkit/NumericEntry/Formatting">Formatting</a></li>
             <li><a href="/maui-toolkit/NumericEntry/Restriction">Restriction</a></li>
+            <li><a href="/maui-toolkit/NumericEntry/Events">Restriction</a></li>
```

Comment:
Use the topic name also Events instead of Restriction

Reply:
updated